### DISCUSSION TITLE
setNewOrder ignores "buildSortQuery"

### DISCUSSION THREAD

**Discussion by michaelLoeffelmann** [freekmurze/eloquent-sortable#161] [Q&A]
Hi just a question:

Is it possible to include the `buildSortQuery` method within the method `setNewOrder`? 
This would allow to use `setNewOrder` also in conjunction with pivot tables (Combined Primary Key).

**Comment by undjike** [comment]
I face the same issue, where I need to set new order for a group.
This is necessary especially on specific columns updates.
